Source/WebInspectorUI/UserInterface/Views/SingleSidebar.js

```diff
@@ -43,8 +43,10 @@ WI.SingleSidebar = class SingleSidebar extends WI.Sidebar
             this.addSubview(this._navigationBar);
         }
 
-        this._resizer = new WI.Resizer(WI.Resizer.RuleOrientation.Vertical, this);
-        this.element.insertBefore(this._resizer.element, this.element.firstChild);
+        this._widthResizer = new WI.Resizer(WI.Resizer.RuleOrientation.Vertical, this);
+        this.element.insertBefore(this._widthResizer.element, this.element.firstChild);
+        
+        this._canMoveToBottom = false;
```

Comment:
nit: instead of setting this to `false` here, i think you might've actually had the right idea originally by having `WI.detailsSidebar._canMoveToBottom = true;` in `Source/WebInspectorUI/Base/Main.js` since that's ultimately the only thing that really can move to the bottom